deploy/auto-install.sh

```
@@ -14,17 +11,17 @@ export YLW='\033[1;33m'
+start_step() { printf '\e[1;34m%-70s\e[m' "$1" && echo "$1" &>>"$LOG_FILE"; }
 report_ok() { echo -e ${GRN}" done"${NON}; }
 report_error() { echo -e ${RED}" error"${NON}; }
 get_env() { grep "^$1" "$2" | cut -d'=' -f 2-50; }
 set_env() {
-	line=$(grep -n "^$1=" $INSTALL_PATH/.env)
+	line=$(grep -n "^$1=" "$INSTALL_PATH/.env")
 	if [ -z "$line" ]; then
-		echo "$1=$2" >>$INSTALL_PATH/.env
+		echo "$1=$2" >>"$INSTALL_PATH/.env"
 	else
 		line_number=$(echo $line | cut -f1 -d:)
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)